benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -89,7 +89,12 @@ public BenchmarkExecutionResult execute(Benchmark benchmark, int benchmarkOrdina
         try {
             macroService.runBenchmarkMacros(benchmark.getBeforeBenchmarkMacros(), benchmark);
 
-            benchmarkExecutionResult = executeBenchmark(benchmark, executionTimeLimit);
+            if (properties.isWarmup()) {
+                benchmarkExecutionResult = warmupBenchmark(benchmark, executionTimeLimit);
```

Comment:
This new `warmup` mode seems to ignore some of the other parameters like `runs`. Would it be possible to set runs to `0` instead, or refactor the API so parameters are independent?

Reply:
After I addressed the other comment, right now we only ignore `runs`. I think it's better than allowing to specify zero runs in the benchmark config and adding yet another parameter.